[Pages] Added limitation for custom domains (#938)
cloudflare/cloudflare-docs: products/pages/src/content/platform/known-issues.md

- pages.dev subdomains currently can't be changed. If you need to change your pages.dev subdomain, delete your project and create a new one.
- Hugo builds automatically run an old version. To run the latest version of Hugo (for instance, 0.80.0), you'll need to set an environment variable. Set `HUGO_VERSION` to `0.80.0` or the Hugo version of your choice.
- By default, we use Node 10 in the Pages build environment. If you need to use a newer Node version, see our [Build configuration page](https://developers.cloudflare.com/pages/platform/build-configuration) for configuration options.

> - It's currently not possible to add a custom domain with wildcard, e.g. *.domain.com.

Reviewer: ```suggestion
- It is currently not possible to add a custom domain with a wildcard, e.g. *.domain.com.
```

> Having an issue that you don't see listed above? Let us know in the Cloudflare Workers Discord! Get your invite at [discord.gg/cloudflaredev](https://discord.gg/cloudflaredev), and share your bug report in the #pages channel.

Reviewer: ```suggestion
If you have an issue that you do not see listed, let the team know in the Cloudflare Workers Discord. Get your invite at [discord.gg/cloudflaredev](https://discord.gg/cloudflaredev), and share your bug report in the #pages channel.
```
